sickrage/helper/common.py

```diff
@@ -328,3 +328,21 @@ def enabled_providers(search_type):
         if x.is_active() and
         hasattr(x, 'enable_{}'.format(search_type)) and
         getattr(x, 'enable_{}'.format(search_type))]
+
+def remove_strings(old_string, unwanted_strings):
+    """
+    Return string removing all unwanted strings on it
+
+    :param old_string: String that will be cleaned
+    :param unwanted_strings: List of unwanted strings
+
+    :returns: the string without the unwanted strings
+    """
+    if not old_string:
+        return
+
+    for item in unwanted_strings:
+        new_string = old_string.replace(item, '') if isinstance(item, str) else new_string
```

Comment:
your not replacing them incrementally.
Just do: old_string = old_string.replace(item, '')

Also don't need to type check there. Your passing the list with strings your self. You can expect it to always have the correct value right?